src/tests/FunctionalTests/iOS/Device/AOT/iOS.Device.Aot.Test.csproj

```diff
@@ -9,6 +9,7 @@
     <ExpectedExitCode>42</ExpectedExitCode>
     <SelfContained>true</SelfContained>
     <UseConsoleUITemplate>true</UseConsoleUITemplate>
+    <HybridGlobalization>true</HybridGlobalization>
```

Comment:
I think this should be assumed with your changes.

Reply:
In FunctionalTests looks like `GlobalizationMode.Hybrid` doesn't return true , I tried adding here, but yes this was unnecessary, still same issue.